doc/tools/docgen_cmd.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+Script to generate documentation for command line utilities
+"""
+import sys
+import re
+from os.path import join as pjoin
+from os import listdir
+from subprocess import Popen, PIPE, CalledProcessError
+
+# version comparison
+from distutils.version import LooseVersion as V
+
+
+def sh3(cmd):
```

Comment:
Oh, I see. It's the third version of a function that runs a command in a shell. We can keep that name and add the link you provided in the Notes section of the docstring.